less/opendev.less

```diff
@@ -12,6 +12,7 @@ img {
 }
 
 table {
```

Comment:
If it presents as my sharing screen, it can't be readable. It will consider as the styling error. 
Personally, I prefer to show as datatable or hide the overflow and enable the scrolling bar.